docker/usr/bin/combine_vcf

```diff
@@ -0,0 +1,180 @@
+#! /usr/bin/python
+"""
+    * The MIT License (MIT)
+    * Copyright (c) 2016 Intel Corporation
+    *
+    * Permission is hereby granted, free of charge, to any person obtaining a copy of
+    * this software and associated documentation files (the "Software"), to deal in
+    * the Software without restriction, including without limitation the rights to
+    * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+    * the Software, and to permit persons to whom the Software is furnished to do so,
+    * subject to the following conditions:
+    *
+    * The above copyright notice and this permission notice shall be included in all
+    * copies or substantial portions of the Software.
+    *
+    * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+    * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+    * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+    * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+    * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+"""
+import sys
+import os
+import os.path
+from datetime import datetime
+from copy import deepcopy
+from subprocess import Popen, call
+import json
+from getopt import getopt
+import uuid
+from collections import OrderedDict
+import vcf
+
+COL_PARTITION_SIZE_UNIT = 16384
+
+loader_cfg_t = {
+    "produce_combined_vcf": True,
+    "num_parallel_vcf_files": 1,
+    "callset_mapping_file" : "",
+    "vcf_output_format" : "z",
+    "vcf_output_filename" : "",
+    "reference_genome" : "", 
+    "column_partitions" : [{"begin": 0, "vcf_output_filename": ""}],
+    "do_ping_pong_buffering" : True,
+    "offload_vcf_output_processing" : True,
+    "produce_GT_field" : False,
+    "size_per_column_partition" : COL_PARTITION_SIZE_UNIT,
+    "vid_mapping_file": "/usr/share/cont-intel/vid.json"    
+}
+
+class CombineVCF():
+    ''' VCF file combiner '''    
+    def __init__(self):
+        self.vcf_inputfiles = None
+        self.callsets = None
+        self.ref_file = None
+        self.output_file = None
+        self.produce_GT_field = None
+
+        DevNull = open(os.devnull, 'wb', 0)
+        if call(["grep", 'docker', '/proc/1/cgroup'], stdout=DevNull, stderr=DevNull) != 0:
+            loader_cfg_t["vid_mapping_file"] = "/home/mingrutar/GenomicsDBPerfTest/dev/templates/vid.json"   
```

Comment:
is there a /home/mingrutar/ directory in the docker container? Maybe it shouldn't be